library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -326,14 +304,159 @@ sealed class Conversation {
             is V1 -> conversationV1.streamDecryptedMessages()
             is V2 -> conversationV2.streamDecryptedMessages()
             is Group -> group.streamDecryptedMessages()
+            is Dm -> dm.streamDecryptedMessages()
+        }
+    }
+
+    // ------- V1 V2 to be deprecated ------
```

Comment:
Organized this so all functionality that will no longer be supported is below this line.